addons/addon-base-raas-ui/packages/base-raas-ui/src/models/users/UsersStore.js

```diff
@@ -33,7 +33,12 @@ const UsersStore = BaseStore.named('UsersStore')
 
     return {
       async doLoad() {
-        const users = (await getUsers()) || [];
```

Comment:
`await getUsers()` needs to be surrounded by a `try, catch` otherwise the error will bubble up and an error screen will be displayed. 
<img width="884" alt="You are not authorized to list users" src="https://user-images.githubusercontent.com/3661906/188057076-b90dce63-b5ce-4aed-a86c-e8d81a8d11af.png">